src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -1620,6 +1620,10 @@ internal enum ErrorCode
         WRN_MissingNonNullTypesContextForAnnotation = 8632,
         WRN_NullabilityMismatchInConstraintsOnImplicitImplementation = 8633,
         WRN_NullabilityMismatchInTypeParameterReferenceTypeConstraint = 8634,
+
+        ERR_ExplicitNonNullTypesAttribute = 8635,
+        WRN_PragmaNonNullTypes = 8636, // PROTOTYPE(NullableReferenceTypes): Remove this fake warning once we have full support for new directive.
```

Comment:
Same here (PROTOTYPE marker)

Reply:
> Same here (PROTOTYPE marker)

Same response

---
In reply to: [223879754](https://github.com/dotnet/roslyn/pull/30387#discussion_r223879754) [](ancestors = 223879754)